src/python/pants/backend/python/goals/export.py

```diff
@@ -109,7 +113,20 @@ async def export_virtualenv(
     return ExportResult(
         f"virtualenv for the resolve '{request.resolve}' (using {min_interpreter})",
         dest,
-        symlinks=[Symlink(venv_abspath, py_version)],
+        digest=pex.digest,
+        post_processing_cmds=[
+            PostProcessingCommand(
+                [
```

Comment:
I'm not sure if it's a concern, but you could avoid the the VenvPex -> Pex + `--include-tools` by also adding the Pex PEX digest and this process can use `["{digest_root}/pex", "{digest_root}/requirements.pex", "venv", ...]` with `{"PEX_MODULE": "pex.tools"}`. Perhaps that allows skipping a PEX re-build.

Reply:
Done, that is probably a perf win.